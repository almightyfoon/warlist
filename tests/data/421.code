c8501b4w4M4M4z4H4HfF4_4U4Wi7__5v4O4M5l4H4H5e5e5f2N4U4W