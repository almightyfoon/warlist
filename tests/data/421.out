https://conflictchamber.com/#c8501b4w4M4M4z4H4HfF4_4U4Wi7__5vmD4MmJ4H4H5e5e5f2N4U4W

Circle Army - 75 / 75 points

[Mohsar 1] Mohsar the Desertwalker [+27]
 - Pureblood Warpwolf [17]
 - Pureblood Warpwolf [17]
 - Wild Argus [7]
 - Woldwyrd [9]
 - Woldwyrd [9]
Swamp Gobber Chef [1]
Tharn Bloodweavers [8]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]
Celestial Fulcrum [19]

Circle Army - 71 / 75 points

[Wurmwood 1] Wurmwood, Tree of Fate [+27]
 - Feral Warpwolf [16]
 - Pureblood Warpwolf [17]
 - Warpwolf Stalker [17]
 - Woldwyrd [9]
 - Woldwyrd [9]
Sentry Stone & Mannikins [5]
Sentry Stone & Mannikins [5]
Shifting Stones [3]
Swamp Gobber Bellows Crew [2]
Wolves of Orboros (max) [11]
 - Wolves of Orboros Chieftain & Standard [4]
